crates/runtime/src/dataaccelerator/duckdb.rs

```diff
@@ -159,6 +182,8 @@ const PARAMETERS: &[ParameterSpec] = &[
     ParameterSpec::component("file"),
     ParameterSpec::runtime("file_watcher"),
     ParameterSpec::component("memory_limit"),
+    ParameterSpec::runtime("connection_pool_size")
```

Comment:
This shouldn't be a parameter that the user configures - Spice should know what value to set here based on how many datasets are attempting to accelerate into the same instance.